MusicNotationKit/MusicNotationKit/Tuplet.swift

```diff
@@ -18,11 +18,17 @@ public struct Tuplet: NoteCollection {
         }
     }
     /// The number of notes of the specified duration that this tuplet contains
```

Comment:
Take out the part of this comment saying "of the specified duration". That is no longer correct. You could probably borrow this for the groupingOrder comment though.